src/org/usfirst/frc/team2706/robot/commands/teleop/Climb.java

```diff
@@ -0,0 +1,31 @@
+package org.usfirst.frc.team2706.robot.commands.teleop;
+
+import org.usfirst.frc.team2706.robot.Robot;
+
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.SpeedController;
+import edu.wpi.first.wpilibj.Talon;
+import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+public class Climb extends Command {
+
+    
+    private boolean finished = false;
+
+    public Climb() {
+    }
+
+    protected void execute(){
+            Robot.climber.Climb();
+    }
+
+    @Override
+    protected boolean isFinished() {
+        return finished;
```

Comment:
This command is intended to run on a whileHeld() trigger. Should those types of commands just go forever (until stopped by the scheduler)? We might eventually add logic to finish once we have the current sensor in place, but otherwise I'm not sure if we need to add a built-in stop condition here.